### PR TITLE
Varvara/feature/reports

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: src/api/reportApi.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// import { AxiosError, AxiosResponse } from "axios";
```
Comment (bezryma) [inline src/api/reportApi.ts]: unused comment)

File: src/api/reportApi.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { instance } from "./axiosInstance";
+
+
+export const reportApi = {
+
+  filterDateToReportVisit: async (data: {
+    type: string,
+    startDate: string,
```
Comment (bezryma) [inline src/api/reportApi.ts]: whether it is possible to take out this attrs to the interface?)

File: src/components/Reports/reports.sass [hunk 3]
```diff
@@ -39,8 +53,100 @@
           font-size: 38px
           color: #fff
 
+// .css-7itnvo-Menu
+//     position: absolute
+//     left: 0
+//     right: 0
```
Comment (bezryma) [inline src/components/Reports/reports.sass:59]: is this block needed?

**Review by bezryma — COMMENTED** [review]
Nice!

File: src/api/reportApi.ts (hunk 2 above)
Comment (varvarashcherbyna7) [reply src/api/reportApi.ts]: yes this is a pertinent remark, I will correct, thank you)

File: src/api/reportApi.ts (hunk 1 above)
Comment (varvarashcherbyna7) [reply src/api/reportApi.ts]: Thanks, I'll delete)

File: src/components/Reports/reports.sass (hunk 3 above)
Comment (varvarashcherbyna7) [reply src/components/Reports/reports.sass:59]: Thanks, I'll delete)

**Pull request merged**